errors.go

```diff
@@ -27,7 +29,8 @@ var (
 
 	// ErrValueThreshold is returned when ValueThreshold is set to a value close to or greater than
 	// uint16.
-	ErrValueThreshold = errors.New("Invalid ValueThreshold, must be lower than uint16")
+	ErrValueThreshold = errors.Errorf(
+		"Invalid ValueThreshold, must be less than %d", math.MaxUint16-16+1)
```

Comment:
I've used `math.MaxUint16-16+1` because of 
https://github.com/dgraph-io/badger/blob/6d3b67d75431ba874d6658c916103070cba3c1f8/db.go#L195-L198